demos/motion_tracking/process_out/process.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.ports.ports import InPort
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.model.py.ports import PyInPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.resources import CPU
+from lava.magma.core.decorator import implements, requires
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+import numpy as np
+
+
+class ProcessOut(AbstractProcess):
```

Comment:
Is this the same as the `Extractor` Process? Can we use that instead or does that create additional overhead for multiple outputs?

Reply:
Any comment on this?